cmd/spire-server/cli/run/run.go

```diff
@@ -99,6 +99,7 @@ type serverConfig struct {
 type experimentalConfig struct {
 	AuthOpaPolicyEngine *authpolicy.OpaEngineConfig `hcl:"auth_opa_policy_engine"`
 	CacheReloadInterval string                      `hcl:"cache_reload_interval"`
+	PruneEventsInterval string                      `hcl:"events_purge_interval"`
```

Comment:
There is a mismatch with the documented setting. I think that this should be `prune_events_interval`.